Sources/FuzzilliCli/main.swift

```diff
@@ -438,7 +438,13 @@ func makeFuzzer(with configuration: Configuration) -> Fuzzer {
     }
 
     // The environment containing available builtins, property names, and method names.
-    let environment = JavaScriptEnvironment(additionalBuiltins: profile.additionalBuiltins, additionalObjectGroups: [])
+    let environment = JavaScriptEnvironment(additionalBuiltins: profile.additionalBuiltins, additionalObjectGroups: profile.additionalObjectGroups)
+    if !profile.additionalBuiltins.isEmpty {
+        logger.verbose("Loaded additional builtins from profile: \(profile.additionalBuiltins.map { $0.key })")
+    }
+    if !profile.additionalObjectGroups.isEmpty {
+        logger.verbose("Loaded additional ObjectGroups from profile: \(profile.additionalObjectGroups.map { $0.name })")
```

Comment:
```suggestion
        logger.info("Loaded additional builtins from profile: \(profile.additionalBuiltins.map { $0.key })")
    }
    if !profile.additionalObjectGroups.isEmpty {
        logger.info("Loaded additional ObjectGroups from profile: \(profile.additionalObjectGroups.map { $0.name })")
```

What are your thoughts on using `logger.info` instead `logger.verbose` here?

Reply:
I don't think this logging is necessary as we will also print all registered builtins in the `initialize` function in the [JavaScriptEnvironment](https://github.com/googleprojectzero/fuzzilli/blob/5f268e1a64f474fac402c79b4f83f7f8c8eeb24a/Sources/Fuzzilli/Environment/JavaScriptEnvironment.swift#L432C9-L433C84). So if you want to keep this for visibility then we should use `.verbose`, otherwise I think we could also drop this, your choice :)

Reply:
Ah, I noticed that `initialize` in JavaScriptEnvironment prints all the registered builtins and objectGroups. Since we fuzz using quite a few Fuzzilli profiles, I think we'd find it helpful to have which of the objectGroups and builtins were loaded from the profile without having to search through the full list of objectGroups and builtins in our logs